HdfsAccessor.go

```diff
@@ -124,7 +124,7 @@ func (this *hdfsAccessorImpl) OpenRead(path string) (ReadSeekCloser, error) {
 	}
 	reader, err2 := client.Open(path)
 	if err2 != nil {
-		//TODO: close connection
+		client.Close()
```

Comment:
Closing only because we won't be reusing it. We create a local client in the method by calling ConnectToNameNode and so the connection would not be reused.